File: csharp/pattenrs/Patterns.cs
using Creational;
using Structural;
using Behavioral;

namespace Patterns
{
    ///
    /// <summary>Class <c>DesignPatterns</c> invokes all  patterns for testing purposes. </summary>
    ///
    class DesignPatterns
    {
        public static void Run()
        {
            Console.WriteLine("\nPatterns");
            CreationalPatterns.Run();
            StructuralPatterns.Run();
            BehavioralPatterns.Run();
        }
    }
}

File: golang/patterns/patterns.go
package patterns

import (
	"fmt"

	"github.com/Tamplier2911/gof-design-patterns/golang/patterns/behavioral"
	"github.com/Tamplier2911/gof-design-patterns/golang/patterns/creational"
	"github.com/Tamplier2911/gof-design-patterns/golang/patterns/structural"
)

func Run() {
	fmt.Println("\nGOF Design Patterns")

	creational.Creational()
	structural.Structural()
	behavioral.Behavioral()
}
